packages/core/src/navigation/utils/index.ts

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Directions, setAttributes, setOrRemoveAttribute } from '@cds/core/internal';
+import { CdsNavigationGroup, CdsNavigationItem, CdsNavigationStart } from '../index.js';
+
+export const NAVIGATION_ITEM_TEXT = 'navigation-item-text';
+export const DEFAULT_NAVIGATION_LAYOUT = 'vertical';
+
+export type FocusableElement = CdsNavigationItem | CdsNavigationStart;
+
+export function expandedGroupEvent(event: KeyboardEvent): boolean {
```

Comment:
I was confused  Reading these helpers in the component code. The name seems like it was calling an event. But all of us really doing is figuring out if an event target is part of an expand event. would it be better to name it something like `isExpandedGroupEvent`? Just so we know to expect a boolean and no other work coming out of it...

Reply:
Updated names to be explicit and use the form `isSomethingSomethingExpanded`.